examples/cms-contentstack/README.md

```diff
@@ -0,0 +1,131 @@
+- Management token (https://www.contentstack.com/docs/developers/create-tokens/generate-a-management-token/) with read permission only
+
+### Step 1. Login to Contentstack CLI
+
+#### Step A Set Login Region
+
+By default, the CLI uses the North American endpoint. To use the European endpoint, run the following command in your terminal (command prompt).
+
+`csdx config:set:region EU`
+
+> Note: For NA region, you can skip [step A].
+
+#### Step B Login in to Contentstack CLI
+
+After setting the region, login in to Contentstack CLI by running the following command:
+
+`csdx auth:login`
+
+### Step 2. Import Stack for Starter Apps
+
+Now, import the stack for starter apps using the seed command. The seed command helps you to easily import and create a stack.
+To start building the starter app, run the following command in your terminal and then follow the on-screen instructions that creates the stack with the required content types, entries and assets.
+
+`csdx cm:seed -r "contentstack/stack-next.js-starter-example"`
+
+### Step 3. Create Delivery and Management token
+
+After importing the stack, navigate to the tokens section in your stack’s Settings/tokens. Create the delivery token and for the live preview feature support, create a new management token.
+
+> Note: We need the management token with READ only access. We recommend you to copy and save the management token as it is visible only once.
+
+### Step 4. Configure the Starter App
+
+Copy the `.env.local.example` file from your root directory to `.env.local` (which is ignored by Git): or use the following command.
+
+```bash
+cp .env.local.example .env.local
+```
+
+Then, set each variable on `.env.local` as follows:
+
+- `CONTENTSTACK_API_KEY` is your Stack API key.
+- `CONTENTSTACK_DELIVERY_TOKEN` is the publishing environment delivery token from step 3.
+
+- `CONTENTSTACK_ENVIRONMENT` is the name of your published environment for the delivery token.
+
+- `CONTENTSTACK_MANAGEMENT_TOKEN` is the stack management token generated in step 3.
+
+- `CONTENTSTACK_API_HOST` is the api host URL.
+  For the NA region use `api.contentstack.io` and for the EU region use `eu-app.contentstack.com`.
+
+- `CONTENTSTACK_APP_HOST` is the app host URL of your stack
+  For the NA region use `app.contentstack.com` and for the EU region you can use ‘eu-app.contentstack.com’.
+
+- `CONTENTSTACK_LIVE_PREVIEW` is to enable and disable live preview from starter apps. For live preview features, always set this value to true.
+
+Your `.env.local` file should look like this:
+
+```bash
+CONTENTSTACK_API_KEY= blt12…
+CONTENTSTACK_DELIVERY_TOKEN=cs123…
+CONTENTSTACK_ENVIRONMENT=production
+
+## For live preview
+CONTENTSTACK_MANAGEMENT_TOKEN= cs123…
+CONTENTSTACK_API_HOST=api.contentstack.io
+CONTENTSTACK_APP_HOST=app.contentstack.com
+CONTENTSTACK_LIVE_PREVIEW=true
+
+```
+
+### Step 5. Run Next.js in the Development Mode
+
+```bash
+npm install
+npm run dev
+# or
+yarn install
+yarn dev
+```
```

Comment:
Could we match other examples here and also add `pnpm`?

Reply:
Thanks for reviewing @balazsorban44 . I will check and add the changes

Reply:
added pnpm installation command